src/dialects/abstract/query-interface.d.ts

```diff
@@ -280,7 +281,7 @@ export interface RemoveColumnOptions extends RemoveColumnQueryOptions, QueryRawO
 * This interface is available through sequelize.queryInterface. It should not be commonly used, but it's
 * referenced anyway, so it can be used.
 */
-export class QueryInterface {
+export class QueryInterface extends AbstractQueryInterfaceTypescript {
```

Comment:
don´t forget to `extends AbstractQueryInterfaceTypescript` in `query-interface.js` too

Reply:
Yep, should've marked this as a draft PR!